templates/NOTES.txt

```diff
@@ -1,28 +1,19 @@
 1. In Helm Version <2.7 following job needs to be cleaned manually.
 
 {{- if .Release.IsUpgrade }}
-  kubectl delete job post-upgrade-{{ template "fullname" . }}-dbmigrate
+  kubectl delete job post-upgrade-{{ template "timed.fullname" . }}-dbmigrate
 {{- end }}
 {{- if .Release.IsInstall }}
-  kubectl delete job post-install-{{ template "fullname" . }}-dbmigrate
+  kubectl delete job post-install-{{ template "timed.fullname" . }}-dbmigrate
 {{- end }}
 
 2. Open following URL to start using Timed:
 {{- if .Values.ingress.enabled }}
 {{- range .Values.ingress.hosts }}
```

Comment:
This `range` seems to have lost its `end`:

```
Error: UPGRADE FAILED: parse error in "timed-chart/templates/NOTES.txt": template: timed-chart/templates/NOTES.txt:12: unexpected EOF
```

Apart from that I seem to have broken my helm setup so I can't currently do a complete test of the changes. I'll do so as soon as I've figured out how to repair a broken helm on ACS.

Reply:
Fixed.